iphone/Classes/TiAppiOSProxy.m

```diff
@@ -407,6 +418,20 @@ - (TiAppiOSBackgroundServiceProxy *)registerBackgroundService:(id)args
   return proxy;
 }
 
+- (void)registerBackgroundTask:(id)args
+{
+  ENSURE_SINGLE_ARG(args, NSDictionary);
+  ENSURE_STRING(args[@"identifier"]);
+  ENSURE_STRING(args[@"type"]);
+
+  if ([TiUtils isIOSVersionLower:@"13.0"]) {
```

Comment:
The Titanium SDK has a current minimum of iOS 13 already, so you can remove this